css/global.css

```diff
@@ -209,8 +221,184 @@ Global Component Styles (buttons, forms, etc.)
 */
 
 /* Buttons */
+/* footer submit button */
+.text-input .button2 {
+	border-radius: 10px;
+	background-color: var(--button2-color);
+	padding: var(--spacing10) var(--spacing40);
+	color: var(--color-white);
+	border: none;
+	min-width: 187px;
+	min-height: 50px;
+	margin: var(--spacing20);
+	font-weight: var(--font-weight-bold);
+	font-size: var(--font21);
+}
 
-/* Forms */
+.text-input .button2:hover {
+	background-color: var(--color-white);
+	color: var(--color-onyx);
+}
+
+/* FOOTER STYLES */
+footer {
+	background-color: var(--color-onyx);
+	color: var(--color-white);
+}
+
+/* Transition and transform for logo and social links for better visual */
+footer a img {
+	transition: all 0.1s ease-in;
+}
+footer a img:hover {
+	transform: scale(110%);
+}
+
+/* footer main is the container for all content except for very bottom links */
+.footer-main {
+	padding: var(--spacing40) 0;
+
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+/* Logo and land acknowledgement button container*/
+.footer-logo-btn {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+/* Land Acknowledgement button */
+.btn-la {
+	background-color: transparent;
+	color: var(--color-white);
+	border: none;
+	margin: var(--spacing40) 0;
+	font-size: var(--font21);
+}
+
+/* hover effect on all footer links */
+.btn-la:hover,
+.footer-main ul li a:hover,
+.bottom-links a:hover {
+	border-bottom: 2px solid var(--color-white);
+}
+
+
+
+/* Form Styles */
+form {
+	text-align: center;
+	padding: var(--spacing20);
+}
+.text-input {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+.text-input input {
+	height: 50px;
+	width: 100%;
+	border-radius: 10px;
+	border: none;
+}
+#newsletter-input {
+	padding-left: 24px;
+}
+
+/* bottom links */
+.bottom-links {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+}
+
+.social-svgs {
+	display: flex;
+	flex-direction: row;
+	column-gap: 20px;
+}
+
+/* MODAL */
+.hidden {
+	display: none;
+}
+
+.fixed {
+	position: fixed;
+}
+/* modal overlay */
+.overlay-la {
+	position: fixed;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	width: 100%;
+	height: 100%;
+	background: rgba(0, 0, 0, 0.5);
+	backdrop-filter: blur(3px);
+	z-index: 1;
+}
+.modal {
```

Comment:
There is something weird going on with the modal and the footer around tablet sizes (it looks good on phone and desktop). It seems the modal is creating some big space one the right side of the footer.

<img width="1179" alt="Screen Shot 2024-01-12 at 12 47 32 PM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/365537c5-8b41-41f0-b5ef-2b05d4cdba45">
<img width="1666" alt="Screen Shot 2024-01-12 at 12 47 47 PM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/b23a5f37-cb4c-4646-a5d9-28beb2045535">